cdap-data-fabric/src/test/java/co/cask/cdap/data2/datafabric/dataset/service/DatasetInstanceHandlerTest.java

```diff
@@ -451,6 +461,15 @@ private HttpResponse makeInstancesRequest(String namespace) throws IOException {
     return HttpRequests.execute(request);
   }
 
+  private ObjectResponse<List<DatasetSpecificationSummary>> getInstancesWithProperties(String namespace,
+                                                                                       Map<String, String> properties)
+    throws IOException {
+    HttpRequest request = HttpRequest.put(getUrl(namespace, "/data/datasets"))
```

Comment:
should this be a post instead?
In DatasetServiceClient.java, it seems like you're making a POST